intranet/static/css/files.scss

```
@@ -131,6 +131,14 @@ table.directory-list {
     }
 }
 
+@media (max-width: 550px) {
```

```suggestion
@media (max-width: 700px) {
```

This is when the search bar drops to a smaller size and approximately when files cards become a single column